crates/fluvio-spu-schema/src/server/fetch_offset.rs

```diff
@@ -113,6 +113,10 @@ pub struct FetchOffsetPartitionResponse {
 
     /// Last readable offset
     pub last_stable_offset: i64,
+
+    /// Last committed consumer offset
+    #[fluvio(min_version = 24)]
+    pub consumer_offset: Option<i64>,
```

Comment:
Hmm.  There is already `FetchConsumerOffsetsRequest` ? Why is needed here?

Reply:
This is only requested when the stream is created, we can also change to request `FetchConsumerOffsetsRequest` there, but it will need 2 requested.

As we were handling consumer offset directly at `start_offset` of `FetchOffsetPartitionResponse`, I kept with it.

Reply:
Let's use `FetchConsumerOffsetsRequest`. otherwise we have to maintain at two different places.  Making 2 requests are fine as this is only at beginning.

Reply:
Seems that `FetchConsumerOffsetsRequest` return all consumers and doesn't allow passing.
Do you still think that is better use it? If yes, should we add a parameter to filter the replica/consumer_id that we want in the request or in memory?

Reply:
I think that is ok use `FetchOffsetsRequest` as it has consumer_id as a parameter already.
Also is filtered by replica and consumer_id.

<img width="874" alt="image" src="https://github.com/user-attachments/assets/2fa6f1ff-e54e-440c-9e33-9d6b8e8bad6e" />

Reply:
Oh. Then need another field in the response? There is no such thing as consumer offset

Reply:
Either this is raw partition offset or applied to consumer only.  So there is no need to another field

Reply:
Before we're just replacing the `start_offset` by the `consumer_offset` when is fetched with a `consumer_id`. But this only worked for `beginning` mode.

I added an optional field called `consumer_offset` to have it instead `start_offset`. 
Then when we resolve the offset using the `consumer_offset` right.

Reply:
> Either this is raw partition offset or applied to consumer only. So there is no need to another field

The problem is, we don't pass the offset mode in the request to the server, so we don't know how to resolve it there.

Reply:
We solve it in the client, so I think it makes sense to return the consumer offset state separately from the start/end of the topic.

Reply:
Problem is that semantics is unclear.   Maybe then logic should be into the SPU consumer processing not at client.

I would expect that when I get offset, it is valid.  SPU should make sure it's always valid

Reply:
Ok, I can change back to just changing the `start_offset`, but sending in the fetch request the offset (begin, end, ...), and resolve it on SPU.
What do you think?

Reply:
It already has following info:
```
  /// First readable offset.
    pub start_offset: i64,

    /// Last readable offset
    pub last_stable_offset: i64,
```

Reply:
I changed to send two requests on stream creation. 
One for topic offset and another one for the consumer_offset.
We're not handling it via forwarding now.